invoice_report/README.rst

```diff
@@ -0,0 +1,50 @@
+'description': """
```

Comment:
@Rona111 What is this?
You need to check the modules first, not only copy paste.
README is completely wrong.